src/modules/bricks/host-bricks/host-bricks.js

```diff
@@ -109,14 +110,14 @@
             } else {
                 clusterStore.getClusterList()
```

Comment:
@gnehapk this else block is not required, if $rootscope.cluster data is not there present and the user is logged in, index.js is handling it https://github.com/Tendrl/ui/blob/master/src/commons/js/index.js#L157

Reply:
@a2batic The getClusterList call inside index.js is called asynchronously. So, if we refresh the host-bricks views, it might possible that the controller's code gets executed before the getClusterList API call fulfilled.  To ensure that it should work correctly, this else block is required.